tests/spec/fs.copyFile.spec.js

```diff
@@ -0,0 +1,52 @@
+var util = require('../lib/test-utils.js');
+var expect = require('chai').expect;
+
+describe('fs.copyFile', function(){
```

Comment:
Let's add `.skip()` to this, since we know we can't pass these tests yet.

https://mochajs.org/#inclusive-tests